walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,49 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
+
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+
+    Collection<String> attendees = request.getAttendees();
+    
+    // build the list of all events to consider while picking a meeting range 
+    ArrayList<TimeRange> allEventRanges = new ArrayList<TimeRange>();
+    for (Event event : events) {
+      // consider the event as long as there is at least one attendee atending it 
```

Comment:
spelling and also this comment is confusing because you're checking attendees for both event and request.  Specify which attendee you are referring to.